update-server-packaging-war/src/main/resources/codenvy/update-server.properties

```diff
@@ -29,3 +29,7 @@ mail.configuration.path=/codenvy/mail.properties
 
 api.endpoint=https://codenvy.com/api
 saas.api.endpoint=https://codenvy.com/api
+
+codenvy_onprem_user_number_report.title=Weekly On-Prem report
+codenvy_onprem_user_number_report.sender=
+codenvy_onprem_user_number_report.receiver=
```

Comment:
You can try find answer on your question here http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline 
In case of git: If somebody add new line to the end of file that doesn't contain empty line that these changes modify old line. You can see here what I mean https://github.com/sleshchenko/Demo_name/commit/8d852b1a8d1286bbb7c2d1474b287e4ac3430ccf